### PR TITLE
refactored unit tests and constants

### PR DESCRIPTION
**Pull request by cjpsycos**

### CODE REVIEW

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java [hunk 1]
```diff
@@ -135,51 +135,33 @@ public void testCreateBfsCodeUniqueConstrains() {
...
                 .setDraft(false)
                 .setPublished(true)
-                .setTitle(generateTranslatedString(String.format("%s-%s", "UPDATED Title", CHF_1.id().getValue())))
-                .setDescription(generateTranslatedString(String.format("%s-%s", "UPDATED Description", CHF_1.id().getValue())))
-                .setOccupations(OccupationFixture.setOf(occupation("BFS-1"), occupation("BFS-2")))
-                .setCompetences(CompetenceFixture.setOf(competence(SPECIALIST, KPZ_1.id()), competence(BASIC, KPZ_2.id())));
+                .setTitle(generateTranslatedString(String.format("%s-%s", "UPDATED Title", CHF_1.getValue())))
+                .setDescription(generateTranslatedString(String.format("%s-%s", "UPDATED Description", CHF_1.getValue())))
+                .setOccupations(OccupationFixture.setOf(occupation(BFS_1), occupation(BFS_2)))
+                .setCompetences(CompetenceFixture.setOf(competence(SPECIALIST, KPZ_1), competence(BASIC, KPZ_2)));
```
Comment (emina-str) [inline application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java]: shouldn't it be also like : 
`.setCompetences(CompetenceFixture.setOf(competenceSpecial(KPZ_1), competenceBasic(KPZ_2)));`

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java [hunk 2]
```diff
@@ -64,15 +64,15 @@ public void tearDown() {
...
-                competenceElement(HLW_3.id(), CompetenceElementType.KNOW_HOW),
-                competenceElement(HLW_3.id(), CompetenceElementType.KNOW_HOW),
-                competenceElement(HWK_1.id(), CompetenceElementType.KNOW_HOW_INDICATOR),
-                competenceElement(HWK_2.id(), CompetenceElementType.KNOW_HOW_INDICATOR),
-                competenceElement(HWK_3.id(), CompetenceElementType.KNOW_HOW_INDICATOR),
-                competenceElement(WSN_1.id(), CompetenceElementType.KNOWLEDGE),
-                competenceElement(WSN_2.id(), CompetenceElementType.KNOWLEDGE),
-                competenceElement(WSN_3.id(), CompetenceElementType.KNOWLEDGE)
+                competenceElement(HLW_1, CompetenceElementType.KNOW_HOW),
+                competenceElement(HLW_3, CompetenceElementType.KNOW_HOW),
```
Comment (emina-str) [inline application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java]: `HLW_2`

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java [hunk 3]
```diff
@@ -156,20 +154,20 @@ public void testDelete() {
...
```
Comment (emina-str) [inline application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java]: `HLW_2`

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java (hunk 2 above)
Comment (cjpsycos) [reply application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java]: fixed

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java (hunk 3 above)
Comment (cjpsycos) [reply application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/competenceelement/CompetenceElementApplicationServiceTest.java]: fixed

File: application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java (hunk 1 above)
Comment (cjpsycos) [reply application/src/test/java/ch/admin/seco/alv/service/competencecatalog/application/chfiche/ChFicheApplicationServiceTest.java]: fixed

**Pull request merged**